.github/actions/setup-environment/action.yml

```diff
@@ -233,6 +233,9 @@ outputs:
   react-app-gtm-container-id:
```

Comment:
I think this should be renamed to react-app-crn-gtm-container-id to reflect that it is CRN only.

Reply:
updated